### PR TITLE
Update postcss.md

### PR DESCRIPTION
**Pull request by Ctannerweb**

### CODE REVIEW

File: ux-dev/postcss.md [hunk 1]
```diff
@@ -18,34 +19,45 @@ To learn about the plugins we choose to use in narwin-pack read through the narw
...
+## How to install
+### Add PostCSS to your project
 
 ```shell
-$npm install --save ember-cli-postcss
+$npm install --save-dev ember-cli-postcss
 ```
 
-###Add narwin-pack
+### Add narwin-pack
```
Comment (ChrisGoodwin) [inline ux-dev/postcss.md]: Use same wording 'to your project'

File: ux-dev/postcss.md [hunk 2]
```diff
@@ -18,34 +19,45 @@ To learn about the plugins we choose to use in narwin-pack read through the narw
...
 
 ```shell
-$npm install --save narwin-pack
+$npm install --save-dev narwin-pack
 ```
 
-Add PostCSS to the `ember-cli-build.js` file on the root of the Ember project.
+Add PostCSS to the `ember-cli-build.js` file on the root of the Ember project with `inline-svg` path options.
```
Comment (ChrisGoodwin) [inline ux-dev/postcss.md]: 'in' the route?

File: ux-dev/postcss.md [hunk 3]
```diff
@@ -18,34 +19,45 @@ To learn about the plugins we choose to use in narwin-pack read through the narw
...
+        ]
       }
-    ]
-  }
-});
+    }
+  });
+
+  return app.toTree();
+};
```
Comment (ChrisGoodwin) [inline ux-dev/postcss.md]: may need to double check that the inline-svg stuff is working correctly.

**Comment by Ctannerweb** [comment]
@danrichman @ChrisGoodwin I want to get this merged soon. Any objections or further review comments?

**Pull request merged**